src/tests/ftest/util/bytes_utils.py

```diff
@@ -0,0 +1,576 @@
+#!/usr/bin/python
+"""
+(C) Copyright 2020 Intel Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+The Government's rights to use, modify, reproduce, release, perform, display,
+or disclose this software are subject to the terms of the Apache License as
+provided in Contract No. B609815.
+Any reproduction of computer software, computer software documentation, or
+portions thereof marked with this legend must also reproduce the markings.
+"""
+from logging import getLogger
+
+
+class ByteUnit(object):
+    # pylint: disable=too-few-public-methods
+    """Defines the unit for a Byte object."""
+
+    VALUES = ("", "K", "M", "G", "T", "P", "E", "Z", "Y")
+
+    def __init__(self, value=None):
+        """Initialize a ByteUnit object with a unit.
+
+        Args:
+            value (str, optional): unit value. Defaults to None.
+        """
+        self._value = None
+        if value is None:
+            self._value = self.VALUES[0]
+        elif value in self.VALUES:
+            self._value = str(value)
+        elif isinstance(value, str) and value.upper()[0] in self.VALUES:
```

Comment:
Yes, `value` could contain non-supported alpha or numeric values, but that is what `and value.upper()[0] in self.VALUES` ensures will not match.  Now `"M123"` would match because we only care about the first letter.  This was included to support allowing inputs like "KB", "MB", "GB" (and lower case versions of "g" or "gb").